src/config/statsd.ts

```diff
@@ -1,9 +1,22 @@
-import {StatsD, StatsCb, Tags} from 'hot-shots';
+import { StatsD, StatsCb, Tags } from 'hot-shots';
+import bunyan from 'bunyan';
+
+const globalTags = {
+  environment:
+    process.env.NODE_ENV === 'test' ? 'test' : process.env.MICROS_ENV,
+  environment_type:
+    process.env.NODE_ENV === 'test' ? 'testenv' : process.env.MICROS_ENVTYPE,
```

Comment:
You have enough `process.env.NODE_ENV === 'test'` in the file that you could probably just create a const `isTest`